client/my-sites/people/team-list/index.jsx

```diff
@@ -3,33 +3,75 @@
  */
 import { localize } from 'i18n-calypso';
 import React from 'react';
+import { QueryClient, QueryClientProvider } from 'react-query';
+import { useDispatch } from 'react-redux';
+import { errorNotice, removeNotice } from 'calypso/state/notices/actions';
+
+const queryClient = new QueryClient();
 
 /**
  * Internal dependencies
  */
-import SiteUsersFetcher from 'calypso/components/site-users-fetcher';
 import Team from './team';
+import useUsers from 'calypso/data/use-users';
+
+function TeamList( props ) {
+	const dispatch = useDispatch();
+	const { site, search, translate } = props;
+	const fetchOptions = {
+		siteId: site?.ID,
+		order: 'ASC',
+		order_by: 'display_name',
+	};
 
-class TeamList extends React.Component {
-	static displayName = 'TeamList';
-
-	render() {
-		const fetchOptions = {
-			siteId: this.props.site && this.props.site.ID,
-			order: 'ASC',
-			order_by: 'display_name',
-			search: this.props.search ? '*' + this.props.search + '*' : null,
-			search_columns: [ 'display_name', 'user_login' ],
-		};
-
-		Object.freeze( fetchOptions );
-
-		return (
-			<SiteUsersFetcher fetchOptions={ fetchOptions }>
-				<Team { ...this.props } />
-			</SiteUsersFetcher>
-		);
+	if ( search ) {
+		fetchOptions.search = `*${ search }*`;
+		fetchOptions.search_columns = [ 'display_name', 'user_login' ];
 	}
+
+	const {
+		data,
+		isLoading,
+		isFetchingNextPage,
+		hasNextPage,
+		fetchNextPage,
+		error,
+		refetch,
+	} = useUsers( fetchOptions );
+
+	React.useEffect( () => {
+		error &&
+			dispatch(
+				errorNotice( translate( 'There was an error retrieving users.' ), {
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **26** times:
`translate( 'There was an error retrieving users' )` *ES Score: 10*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/try/users-react-query-experiment)